playbook/app/pb_kits/playbook/pb_card/docs/_card_swift.md

```diff
@@ -0,0 +1,30 @@
+### Description
+Card component can easily wrap custom content using the same Playbook pattern.
+
+### When to Use
+- Render content with same visual style and pattern
+- Organize content with Playbook separator style 
+- Render content with Playbook background colors
+
+### When not to Use
+- To execute actions and for tappble areas, try to use buttons instead.
+
+### How to use
+| Code | Component |
```

Comment:
I talked to Colin and he suggested removing the header for the "how to use" table.